src/OpenTelemetry.Exporter.Geneva/GenevaMetricExporterOptions.cs

```diff
@@ -34,7 +35,28 @@ public class GenevaMetricExporterOptions
     /// <summary>
     /// Gets or sets the metric export interval in milliseconds. The default value is 20000.
     /// </summary>
-    public int MetricExportIntervalMilliseconds { get; set; } = 20000;
+    public int MetricExportIntervalMilliseconds
+    {
+        get
+        {
+            return this._metricExporterIntervalMilliseconds;
+        }
+
+        set
+        {
+            if (value < 0)
+            {
+                throw new ArgumentException($"MetricExportIntervalMilliseconds: {value} should be a greater than 0 integer.");
```

Comment:
Consider to use `ArgumentOutOfRangeException`. It inherits from `ArgumentException` but it is more descriptive.

Reply:
Follow https://github.com/open-telemetry/opentelemetry-dotnet-contrib/blob/dcf0156af8697c0f4d95e4fdae8d3b4b778e4810/src/OpenTelemetry.Extensions.AzureMonitor/ApplicationInsightsSampler.cs#L37.